static/js/admin/scannerqueryresult.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener('DOMContentLoaded', () => {
+  if (!document.body.matches('.change-list.model-scannerqueryresult')) {
+    // This is only for the scanners query result change list page.
+    return;
+  }
+
+  let current_addon = {
+    cell: null,
+    link: null,
+  };
+
+  for (row of result_list.tBodies[0].rows) {
```

Comment:
Where is `result_list` coming from?

Reply:
Uh, there used to be a selector to get that element by id... I must have deleted it by accident... but I was saved by the fact that browsers auto-create the global variable since it's the exact id... I'll add an explicit call.

Reply:
https://github.com/mozilla/addons-server/pull/15576/commits/3f0ce7b9fa0ea3e5f205fee1d9ab14a267485beb